site-src/v1alpha2/references/cross-namespace-references.md

```diff
@@ -0,0 +1,273 @@
+# Cross namespace references and ReferencePolicy
+
+In the Gateway API, it is possible to have references between objects cross namespace boundaries.
+In particular, it's expected that Gateways and Routes may exist in different namespaces,
+or that Services may be referred to by Routes in a another namespace.
```

Comment:
> that Gateways and Routes may

ReferencePolicy is not expected to be applied to Gateway-Route references so it would be best to avoid mentioning that here.